optuna/study.py

```diff
@@ -249,8 +249,8 @@ def set_system_attr(self, key, value):
 
         self.storage.set_study_system_attr(self.study_id, key, value)
 
-    def trials_dataframe(self):
-        # type: () -> pd.DataFrame
+    def trials_dataframe(self, show_internal_fields=False):
```

Comment:
In my thought,  the word "show" is typically used for specifying to output something to the standard output or logs, etc.
So I prefer another word (e.g., "include_...") to "show_...".

Reply:
I agree with you. I'll fix it. Thanks!